apps/rush-lib/src/logic/taskExecution/ProjectLogWritable.ts

```diff
@@ -28,24 +27,28 @@ export class ProjectLogWritable extends TerminalWritable {
       this._project.packageName
     );
 
-    this._buildLogPath = path.join(this._project.projectFolder, `${unscopedProjectName}.build.log`);
-    this._errorLogPath = path.join(this._project.projectFolder, `${unscopedProjectName}.build.error.log`);
+    this._logPath = `${this._project.projectFolder}/${unscopedProjectName}.${logFilename}.log`;
+    this._errorLogPath = `${this._project.projectFolder}/${unscopedProjectName}.${logFilename}.error.log`;
```

Comment:
Since we are increasing the number of log filenames by an order of magnitude, maybe it's time to rethink the location of these files? For example, move them to `<projectName>/temp` or `common/temp/logs/<projectName>`.

Reply:
Moved them to `<projectName>/rush-logs`